test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1540,7 +1507,7 @@ def test_cgroup_execjob_end_should_delete_cgroup(self):
         self.server.manager(MGR_CMD_SET, HOOK, attr, self.hook_name)
         self.server.expect(NODE, {'state': 'free'})
         j = Job(TEST_USER)
-        j.set_execargs('/bin/sleep', '1')
+        j.set_sleep_time('1')
```

Comment:
Can set_sleep_time() take a string instead of a number?

Reply:
yes either works. I have changed it to number.